<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NameAndSite</class>
 <widget class="QWidget" name="NameAndSite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Surname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Site:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
